pkg/aws/ec2/api/wrapper.go

```diff
@@ -360,10 +388,12 @@ func prometheusRegister() {
 			ec2modifyNetworkInterfaceAttributeAPICallCnt,
 			ec2modifyNetworkInterfaceAttributeAPIErrCnt,
 			ec2APICallLatencies,
-			vpcCniLeakedENICleanupCnt,
-			vpcrcLeakedENICleanupCnt,
 			ec2DisassociateTrunkInterfaceCallCnt,
 			ec2DisassociateTrunkInterfaceErrCnt,
+			VpcCniAvailableENICnt,
+			VpcRcAvailableENICnt,
+			LeakedENIClusterCleanupCnt,
+			LeakedENINodeCleanupCnt,
 		)
```

Comment:
Can you also add a counter for disassociate call?

Reply:
Metrics for disassociate call is already added in a previous PR #372